Intersect (Core)/Config/PlayerOptions.cs

```diff
@@ -96,6 +101,11 @@ public partial class PlayerOptions
         /// </summary>
         public bool SkipCharacterSelect { get; set; } = false;
 
+        /// <summary>
+        /// When enabled, 'Auto-target switching' is limited to entities within the front view of players.
+        /// </summary>
+        public bool TargetAccountDirection { get; set; } = false;
+
```

Comment:
This is not a new feature, the ability to limit `Auto-target switching` was previously not exposed as an option, here i've just exposed it to the server options. i can undo this and restore the TODO if that helps to move on with this PR.

Reply:
i would honestly rename the second one for something like "`LimitAutoTargetSwitch`" ; this is essentially a feature that's disabled by default and is not exposed anywhere to be configurable, i assume that it wasn't fully finished or something, and yeah, idk if it's taking diagonal movement into account.

this somehow works pretty well if someone wished to further limit not only the ability to automatically turn around towards targets but also, to limit where players can click/tab/switch their current target to.


(shortcut would be to revert this and hide the registry boolean back again since there's already a way to limit the auto turn to target) this is like a trabalenguas (i know, my bad for this ; not really sure what we should do about this _hidden feature_ ).

Reply:
```csharp
        /// <summary>
        /// When enabled, 'Auto-turn to target' is limited to entities within the front view of players.
        /// </summary>
        public bool LimitAutoTurnToTarget { get; set; } = false;
        
        /// <summary>
        /// When enabled, selecting a target is limited to entities within the front view of players.
        /// </summary>
        public bool LimitTargetSelect { get; set; } = false;
```

Reply:
decided to remove the second one, this was an unfinished and hidden logic and is not used at all now.